addons/delivery/models/stock_picking.py

```diff
@@ -147,6 +154,33 @@ def put_in_pack(self):
         }
 
     @api.multi
+    def action_send_confirmation_email(self):
+        self.ensure_one()
+        self._get_tracking_url()
+        delivery_template = self.env.ref('delivery.delivery_confirmation_mail_template', False)
+        compose_form = self.env.ref('mail.email_compose_message_wizard_form', False)
+        ctx = dict(
+            default_model='stock.picking',
+            default_res_id=self.id,
+            default_use_template=bool(delivery_template),
+            default_template_id=delivery_template.id,
+            default_composition_mode='comment',
+            custom_layout='delivery.delivery_order_mail_template_data_notification_email'
+        )
+        self.is_delivery_send_mail = False
```

Comment:
No. If you don't sent the email with the wizard. The picking will be marked as 'mail sent' and it's absolutely not the case.